dnf/CMakeLists.txt

```diff
@@ -5,6 +5,11 @@ glib_compile_resources (DNF_COMMAND_INSTALL plugins/install/dnf-command-install.
                         INTERNAL)
 list (APPEND DNF_COMMAND_INSTALL "plugins/install/dnf-command-install.c")
 
+glib_compile_resources (DNF_COMMAND_INSTALL plugins/remove/dnf-command-remove.gresource.xml
```

Comment:
This line rewrites DNF_COMMAND_INSTALL variable. "install" plugin is not compiled. Use another variable DNF_COMMAND_REMOVE and add it to add_executable.